packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -14,8 +14,9 @@ const utils = require('../../lib/utils');
 const appGenerator = path.join(__dirname, '../../generators/app');
 const rootDir = path.join(__dirname, '../../../..');
 const sandboxDir = path.join(rootDir, 'sandbox');
+const {skipIf} = require('@loopback/testlab');
 
-describe('app-generator (SLOW)', function () {
+skipIf(process.env.CI == null, describe, 'app-generator (SLOW)', function () {
```

Comment:
Please add a code comment to explain what's happening here, why are we doing that and how can users run this test locally.